web/components/ui/Header/Header.tsx

```diff
@@ -23,7 +23,9 @@ export const Header: FC<HeaderComponentProps> = ({
       <div id="header-logo">
         <OwncastLogo variant="contrast" />
       </div>
-      <span id="global-header-text">{name}</span>
+      <h1>
+        <span id="global-header-text">{name}</span>
```

Comment:
I would venture to guess that for accessibility it might be better for the content to be directly inside the new h1 tags and not again wrapped with unnecessary tags. You might have to change the styling of the h1 tag to be `inline` but I'm just guessing. Edit the corresponding `.module` file to make any necessary styling changes if needed.